src/pkg/packager/prepare.go

```diff
@@ -356,6 +377,28 @@ func (p *Packager) processUnstructuredImages(resource *unstructured.Unstructured
 	return matchedImages, maybeImages, nil
 }
 
+func (p *Packager) findWhyResources(resources []*unstructured.Unstructured, whyImage, componentName, resourceName string, isChart bool) []string {
+	output := []string{}
+	for _, resource := range resources {
+		bytes, _ := resource.MarshalJSON()
```

Comment:
We should handle the error returned from `resource.MarshalJSON()`

Reply:
correct, seems like I missed this one as well